dexcom/fetch/translate.go

```diff
@@ -388,8 +376,26 @@ func translateEGVToDatum(egv *dexcom.EGV, unit *string, rateUnit *string) data.D
 				"threshold": dexcom.EGVValuePinnedMgdLMaximum,
 			}}
 		}
+	case dexcom.EGVUnitMmolL:
+		if *datum.Value < dexcom.EGVValuePinnedMmolLMinimum {
+			datum.Value = pointer.FromFloat64(dexcom.EGVValuePinnedMmolLMinimum - 0.1) //check this
+			datum.Annotations = &metadata.MetadataArray{{
+				"code":      "bg/out-of-range",
+				"value":     "low",
+				"threshold": dexcom.EGVValuePinnedMmolLMinimum,
+			}}
+		} else if *datum.Value > dexcom.EGVValuePinnedMmolLMaximum {
+			datum.Value = pointer.FromFloat64(dexcom.EGVValuePinnedMmolLMaximum + 0.1) //check this
```

Comment:
seeking feedback on this - just went for arbitory minimal value

Reply:
Seems reasonable.